src/typings/utils/Class.ts

```diff
@@ -0,0 +1,10 @@
+// Copied from 'type-fest' (https://github.com/sindresorhus/type-fest/blob/main/source/basic.d.ts)
+
+import type { Constructor } from "./Constructor";
+
+/**
+Matches a [`class`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes).
+*/
+export type Class<T = unknown, Arguments extends unknown[] = any[]> = Constructor<T, Arguments> & {
```

Comment:
Why renaming this? It's unneeded breaking change. You were supposed to do simple fast maintenance repo tweaks, not refactor the codebase.

Reply:
Is it used externally? It looks internal to me